### DISCUSSION TITLE
Is xDrip fully functional on Android 15?

### DISCUSSION THREAD

**Discussion by Gonzmdv** [NightscoutFoundation/xDrip#3825] [General]
Hello, my Pixel 6 Pro is asking me permission to install Android 15. Is xDrip fully functional on Android 15 to use it with Dexcom G6/G7?
Thank you!

**Comment by Navid200** [comment]
It is for me on Pixel 6a.  I have updated to Android 15 when it came out a few months ago and have no issues with G7.
However, there are individuals who are experiencing problems with it:
https://github.com/NightscoutFoundation/xDrip/discussions/3733
